### ISSUE TITLE
Try to get added to Mink's driver documentation

### ISSUE THREAD

**Issue by tuxayo** [jcalderonzumba/MinkPhantomJSDriver#52]
http://mink.behat.org/en/latest/guides/drivers.html

At least get mentioned as an additional driver.

Also when reading that MinkPhantomJSDriver supports
> ALL of the features defined in Mink DriverInterface. maximizeWindow is the only one not implemented since is a headless browser it does not make sense to implement it.

Should we understand that it's column would be full of yes expect for the "Window maximizing" ?
(just in case that the readme would be outdated)

**Comment by jcalderonzumba** [comment]
Hi @tuxayo yes, i have already started a talk with Mink people but the driver needs to finish some recommendations coming from the before we get added to it. 

See https://github.com/jcalderonzumba/MinkPhantomJSDriver/pull/50, is a WIP that i have not yet have the time to finish. Once that is properly done i can ping them again so we can be added.

**Comment by tuxayo** [comment]
Right, I haven't seen that it was in the process to be integrated to Mink itself, it's even better! :-D
Is there some grunt work that external people could do to offload you a bit on this task?

**Comment by jcalderonzumba** [comment]
@tuxayo The work is mostly to pass all the mink test they have defined, because there are some that as of today do not pass due to phantomjs limitations.

**Comment by PatchRanger** [comment]
@jcalderonzumba As far as I can see there are only 2 failed tests: https://travis-ci.org/jcalderonzumba/MinkPhantomJSDriver/jobs/251892172 - and one of them is going to be fixed by https://github.com/minkphp/driver-testsuite/pull/12 . `MinkSelenium2Driver` skips PhantomJS tests, marking them as `allow_failure: true`: https://github.com/minkphp/MinkSelenium2Driver/blob/master/.travis.yml - that implementation has plenty of bugs: https://travis-ci.org/minkphp/MinkSelenium2Driver/jobs/251887708 .

@jcalderonzumba Please insist to be added to the list - as your implementation looks much cleaner.

**Comment by jcalderonzumba** [comment]
I'll try, let's first get your PR merged 👍
